contracts/Rollup.sol

```diff
@@ -151,8 +158,9 @@ contract Rollup is Ownable, RollupHelpers, RollupInterface {
         require(msg.value >= feeAddToken, 'Amount is not enough to cover token fees');
         uint tokenId = tokens.push(tokenAddress) - 1;
         tokenList[tokenId] = tokenAddress;
+        feeTokenAddress.transfer(msg.value);
         // increase fees for next token deposit
-        feeAddToken = feeAddToken * 2;
+        feeAddToken = feeAddToken / 4 + feeAddToken;
```

Comment:
I would add brackets to make operations clear
`feeAddToken = (feeAddToken / 4) + feeAddToken;`